icp_scan_matcher_node:
  ros__parameters:
    icp_scan_matcher:
      topic_name:
        # cloud: "/camera/depth_registered/points"
        cloud: "/tugbot/points"
        scan: "/scan"
        imu_pose: "/wit_ros/imu_pose"
      tf_frame:
        map_frame: "map"
        robot_frame: "base_link"
      2d: false
      broadcast_period: 0.001
      # 点群パラメータ
      min_point_cloud_size: 30
      min_score_limit: 0.1
      filter:
        voxelgrid_size: 0.2
        target_update_min_score: 0.001
      normal_estimation:
        radius_search: 0.7
